binding/Binding/SKData.cs

```diff
@@ -103,7 +103,7 @@ public static SKData Create (Stream stream)
 			if (stream == null)
 				throw new ArgumentNullException (nameof (stream));
 			if (stream.CanSeek) {
-				return Create (stream, stream.Length);
+				return Create (stream, stream.Length - stream.Position);
```

Comment:
This was a bug I found when you pass a stream that was not at 0. This can be used to create a data object from a section of a stream.